sdks/java/io/elasticsearch-tests/elasticsearch-tests-2/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOIT.java

```diff
@@ -41,12 +41,14 @@
  * <p>This test requires a running instance of Elasticsearch, and the test dataset must exist in the
  * database.
  *
- * <p>You can run this test by doing the following from the beam parent module directory:
+ * <p>You can run this test by doing the following from the beam parent module directory with the
+ * correct server IP:
  *
  * <pre>
- *  mvn -e -Pio-it verify -pl sdks/java/io/elasticsearch -DintegrationTestPipelineOptions='[
- *  "--elasticsearchServer=1.2.3.4",
- *  "--elasticsearchHttpPort=9200"]'
+ *  mvn -e -Pio-it verify -pl elasticsearch-tests/elasticsearch-tests-2 \
+ *    -DintegrationTestPipelineOptions='[
+ *      "--elasticsearchServer=127.0.0.1",
```

Comment:
ES integ server is not in the build server, don't give the impression that it is, 1.2.3.4 is better